src/main/java/com/abcdedu_backend/board/BoardController.java

```diff
@@ -53,22 +53,14 @@ public Response<Void> deleteBoard(@PathVariable Long boardId, @JwtValidation Lon
     }
 
     @Operation(summary = "모든 게시판 카테고리 조회", description = "카테고리를 모두 조회합니다.")
-    @GetMapping("/")
+    @GetMapping()
```

Comment:
p5: 이거는 이렇게 지우는게더 깔끔할것같네요!
```suggestion
    @GetMapping
```

Reply:
오 그러네요 ㅋㅋ 반영했숩니다~ 감사해용